core/sdl/dreamconn.cpp

```diff
@@ -372,14 +372,27 @@ bool DreamConnGamepad::gamepad_axis_input(u32 code, int value)
 {
 	if (!is_detecting_input())
 	{
+	// For DreamcastControllerUsb, we need to diff between Linux which exposes the UDB HID event codes as expected, and MacOS/Windows which require a dedicated mapping
+	// Change is temporary for testing - will be superseded by an extension of SDL providing default mappings for DreamcastControllerUsb
```

Comment:
Ideally a custom SDL mapping for the DreamcastUSB bindings should be loaded at start up with `SDL_GameControllerAddMapping`. In the mean time `leftTrigger` and `rightTrigger` should be changed to 2 and 5 resp. when a DreamcastUSB is detected since they are known. They are used elsewhere in the code so they should match the actual controller.

Reply:
Thanks for confirming, thats what I had in mind as well

Reply:
Added a fix for leftTrigger/rightTrigger and reverted the changes in `gamepad_axis_input()`

Reply:
Added mappings for Dreamcast Controller USB during `input_sdl_init()`